vrtUtil/ImageDiff.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs');
+const { PNG } = require('pngjs');
+const pixelmatch = require('pixelmatch');
```

Comment:
yeah this file was being referenced to by the testcafe config file before when we were using the `customActions` feature and it didn't support that syntax.  But it has since been refactored and no longer referenced there so this can be changed back to use `import` 👍